extensions/grpc/deployment/src/main/java/io/quarkus/grpc/deployment/GrpcServerProcessor.java

```diff
@@ -64,7 +64,7 @@ void discoverBindableServices(BuildProducer<BindableServiceBuildItem> bindables,
     @BuildStep(onlyIf = IsNormal.class)
     public KubernetesPortBuildItem registerGrpcServiceInKubernetes(List<BindableServiceBuildItem> bindables) {
         if (!bindables.isEmpty()) {
-            int port = ConfigProvider.getConfig().getOptionalValue("quarkus.grpc-server.port", Integer.class)
+            int port = ConfigProvider.getConfig().getOptionalValue("quarkus.grpc.server.port", Integer.class)
```

Comment:
We are in a build step. Why don't we inject the configuration object to avoid this sort of issues?

Reply:
Because that creates a cycle. Vert.x HTTP does the same for the HTTP port.

If we use the configuration, the augmentation fails with:
```
Caused by: java.lang.IllegalArgumentException: Run time configuration cannot be consumed here unless the method is a @Recorder at io.quarkus.grpc.runtime.config.GrpcConfiguration conf of public io.quarkus.kubernetes.spi.KubernetesPortBuildItem io.quarkus.grpc.deployment.GrpcServerProcessor.registerGrpcServiceInKubernetes(java.util.List,io.quarkus.grpc.runtime.config.GrpcConfiguration) of class io.quarkus.grpc.deployment.GrpcServerProcessor
```

Reply:
Ah yes, forgot about this issue: we are consuming runtime configuration at build time. I withdraw my comment.